contracts/token/psp1155/src/traits.rs

```diff
@@ -324,7 +326,9 @@ pub trait PSP1155: PSP1155Storage {
                     }
                 }
             }
-        }
+        }?;
```

Comment:
You should do load even you get `Error`. So you need to save a result, do load, and after return a result

Reply:
The same in another places